chcemvediet/apps/inforequests/tests/test_models/test_action.py

```diff
@@ -573,7 +641,7 @@ def test_send_by_email_collected_recipients(self):
                 u'Request Bcc <request-bcc@a.com>',
                 # Inbound email contributes with its from address only
                 u'Refusal From <refusal-from@a.com>',
-                # Currect obligee addresses
+                # Current obligee addresses
```

Comment:
Asi to malo byt skor "Correct", lebo neriesime tu ziadne non-current emaily

Reply:
:ok:

Reply:
@viliambalaz A sorry, mal si pravdu, ma to byt `Currect`. Vo vyzname, ze emailove adresy v obligees sa mozu casom menit. A test kontroluje to, ze v `recipient_set` je zjednotenie vsetkych obligee adries, ktore pocas zivota ziadosti boli. Teda, ze su to adresy, na ktore sme poslali prvotnu ziadost `request-*@a.com`, adresa z ktorej nam prisla odpoved `refusal-from@a.com`, ale aj adresy, ktore su v obligee objekte nastavene teraz `oblige*@a.com`

Reply:
:ok: